graphql/environment-nodes-query-4.graphql

```diff
@@ -0,0 +1,17 @@
+query {
+    environmentNodes(filter: { nodeName: "JDP" }) {
+        descendantTargets {
+            snapshot {
+                name
+                state
+            }
+            recordings {
+                active {
+                    archive {
```

Comment:
I was looking at this example and mistakenly confused the keywords `archived` (query) and `archive` ("mutation") with each other. What do you think about making the mutation keywords easier to identify, or at least change `archive` to `archiveRecording` to match `startRecording`?

Reply:
Yea, I agree. I made the same mistake and thought it was a bug when I demoed this to you. I do think it would be better to have some kind of naming convention that marks mutations. My first thought was to just start them with `do`, in this case `doArchive`, but that gets pretty weird for `doStart` or `doStartRecording`. I think it would be nice to have some kind of uniform prefix/postfix for all mutations still, so they can be identified very quickly at a glance. Maybe `doStartRecording` is okay if it's also accompanied by `doArchiveRecording` and `doDeleteRecording`.